integration-libs/opf/base/occ/config/default-occ-opf-base-config.ts

```diff
@@ -6,12 +6,11 @@
 
 import { OccConfig } from '@spartacus/core';
 
-export const defaultOccOpfConfig: OccConfig = {
+export const defaultOccOpfBaseConfig: OccConfig = {
   backend: {
     occ: {
       endpoints: {
         getActiveConfigurations: 'active-configurations',
```

Comment:
[MAJOR]
Endpoints that are not present in the OCC backend (but only in the Workbench backend), should not be stored at the `backend.occ` key, but at the `backend.opf` key.
Likewise, the Adapters calling those endpoints should not be prefixed with `Occ` (e.g. `OccOpfAdapter`), because they have nothing to do with Occ.
Similarly, there code for calling those endpoints should not be stored at the `/occ` folder (and sub-library name), but a different name.

TBH I'm having hard time proposing a different name, because I'm lacking knowledge on the nature of the OPF backend. Is it `OPF backend`? `OPF Workbench backend`? ...? When we come up with the _name of the backend_, we should use it consistently for places referring to this backend, instead of the term `OCC`.

Note: As far as I can udnerstand, those endpoints are for example `/active-configurations`, `/payments/get-cta-scripts` etc. - they are not OCC endpoints.
But `/cart/accessCode` and `placePaymentAuthorizedOrder` are OCC Commerce's endpoints (so their logic is accurate being put within OCC-prefixed places.

Reply:
I clarified with the Bamboo team that all endpoints in a Commerce Cloud Adapter will only affect OPF functionality and they will be never a part of the OCC.

So my proposal is to change a name into:
```
    "opf-api" {
      endpoints: {
        getActiveConfigurations: 'active-configurations',
```

and the same for the folder "opf-api"
We can also use such prefix "OpfApi..." for all files inside the folder, replacing the "Occ" part. What do you think?